connectors/es/__init__.py

```diff
@@ -6,4 +6,6 @@
 from connectors.es.client import ESClient  # NOQA
 from connectors.es.document import ESDocument, InvalidDocumentSourceError  # NOQA
 from connectors.es.index import ESIndex  # NOQA
-from connectors.es.settings import DEFAULT_LANGUAGE, Mappings  # NOQA
+
+TIMESTAMP_FIELD = "_timestamp"
+DEFAULT_LANGUAGE = "en"
```

Comment:
These constants were the only pieces from `settings.py` that seemed worth keeping.

We could go farther and remove `language` as a concept from connectors as well. However, this would require API changes and a change to the Protocol (`"language"` field in the connector record), as well as a migration to remove it, so I'm opting to keep that vestigial for now.